CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ set(DRIVER_PACKAGE_NAME "scap")
 
 include(CheckSymbolExists)
 check_symbol_exists(strlcpy "string.h" HAVE_STRLCPY)
-if(HAVE_STRLCPY)
+if(HAVE_STRLCPY AND NOT APPLE)
```

Comment:
This is better but why doesn't the check work on mac os?

Reply:
this is fixed in 3208ea42f8ed89a01a178764cf92aed643d5e6b5, as recommended by @LucaGuerra